config/remote/remote_test.go

```diff
@@ -356,9 +384,11 @@ func TestGetUpdatedTelioConfig(t *testing.T) {
 	assert.NotNil(t, info3inc2)
 
 	// files are modified on disk - should be greater time
-	assert.Greater(t, info3.ModTime().Nanosecond(), info1.ModTime().Nanosecond())
-	assert.Greater(t, info3inc1.ModTime().Nanosecond(), info1inc1.ModTime().Nanosecond())
-	assert.Greater(t, info3inc2.ModTime().Nanosecond(), info1inc2.ModTime().Nanosecond())
+	// sometimes I/O operations can get delayed, thus here we use active-waiting approach bounded by the timeout
+	timeout := 2 * time.Second
+	assertEventuallyGreater(t, func() int { return info3.ModTime().Nanosecond() }, func() int { return info1.ModTime().Nanosecond() }, timeout)
```

Comment:
[small] maybe this could be DRYied a bit with higher level func like below?
```
 ...
	assertEventuallyGreater(t, modTimeNanos(info3), modTimeNanos(info1), timeout)
 ...
func modTimeNanos(fi os.FileInfo) func() int {
	return func() int { return fi.ModTime().Nanosecond() }
}
```

Reply:
Fixed.